### DISCUSSION TITLE
spring-projects/spring-data-aerospike is now 💀👻

### DISCUSSION THREAD

**Discussion by rbotzer** [aerospike/spring-data-aerospike#524] [Team Posts]
Congratulations, the original repo gave up the ghost for the Aerospike Community one: https://github.com/spring-projects/spring-data-aerospike.

However, this broke the [Getting Help](https://github.com/aerospike-community/spring-data-aerospike#getting-help) section. The links to the User Guide, JavaDocs, home page. In your infinite spare time, please look into fixing those.

**Comment by Aloren** [comment]
Will work on this. Thank you.
